app/routers/playlistVideo.py

```diff
@@ -0,0 +1,41 @@
+from db.database import get_db
+from sqlalchemy.orm import Session
+from fastapi import status, Depends, APIRouter
+from utils import OAuth2
+from sqlalchemy import exc
+import schemas.video, schemas.playlist
+import models.video, models.playlist, models.playlistVideo
+
+router = APIRouter(
+    prefix="/addVideos",
+    tags = ['addVideos']
+)
+
+@router.post("/{id}",status_code=status.HTTP_201_CREATED)
+def addVideoPlaylist(id: int,VideoPlaylist:schemas.video.VideoPlaylistOut, db: Session = Depends(get_db),current_user: int = Depends(OAuth2.get_current_user)):
```

Comment:
```suggestion
def addVideoPlaylist(id: int, videoPlaylist: schemas.video.VideoPlaylistOut, db: Session = Depends(get_db), current_user: int = Depends(OAuth2.get_current_user)):
```

wront schemas, it can't be out